api/buildings/admin.py

```diff
@@ -29,6 +30,50 @@ class BuildingAdmin(admin.ModelAdmin):
         "general_id",
     )
     search_fields = ("address",)
+    actions = (
+        "make_choose",
+        "make_accepted",
+        "make_rejected",
+    )
+
+    def make_choose(self, request, queryset):
```

Comment:
I think `pending` sounds better than `choose` in this

Reply:
I know, but the building model has these statuses [choose, accepted, rejected]. It's ok if I change `choose` with `pending ` here and in the Building model field?

Reply:
Sure, you can go ahead with this change.

Reply:
Ok, thanks!